src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/ApplicationManifestKindValueProvider.cs

```diff
@@ -31,6 +31,11 @@ public ApplicationManifestKindValueProvider(ITemporaryPropertyStorage temporaryP
             _temporaryPropertyStorage = temporaryPropertyStorage;
         }
 
+        public override Task<bool> IsValueDefinedInContextAsync(string propertyName, IProjectProperties defaultProperties)
+        {
+            return IsValueDefinedInContextMsBuildPropertiesAsync(defaultProperties, new [] { NoManifestMSBuildProperty, ApplicationManifestMSBuildProperty });
```

Comment:
You can pull the `new [] { NoManifestMSBuildProperty, ApplicationManifestMSBuildProperty }` out into a private static field so we're not creating a new array every time this is called. Ditto for the other interceptors.